snapcraft/internal/build_providers/_multipass/_windows.py

```diff
@@ -0,0 +1,141 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2018 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import os.path
+import requests
+import shutil
+import subprocess
+import sys
+import tempfile
+
+from snapcraft.internal.errors import (
+    SnapcraftWindowsMultipassDownloadFailed,
+    SnapcraftWindowsMultipassInstallationFailed,
+)
+
+if sys.platform == "win32":
+    import winreg
+
+
+logger = logging.getLogger(__name__)
+
+
+_MULTIPASS_WIN32_INSTALLER_URL = "https://github.com/CanonicalLtd/multipass/releases/download/v0.8.0/multipass-0.8.0+win-win64.exe"
```

Comment:
@Saviq do we have a generic link for this that points to the latest release?

Reply:
I'm afraid not. Is this something users will go to with their browser, or are you trying to download and install programatically?

Reply:
Right, I can see now that you're downloading it yourself. Could you maybe parse it out of https://api.github.com/repos/CanonicalLtd/multipass/releases/latest ?